File: primerTrimestre/java basico/1secuenciales/Ej5sec.java
/**Escribir un programa que convierta un valor dado en grados Fahrenheit a grados Celsius.
 * 
 *@author Antonio Quesada 
 */

 

import java.util.Scanner;

public class Ej5sec {
  public static void main(String[] args) {
    Scanner s = new Scanner(System.in);
    
    //Definimos las variables.
    double celsius;
    double fahrenheit;
    
    //Leemos el valor en grados celsius.
    System.out.println("Introduce el valor en grados celsius:");
    celsius = s.nextDouble();
    
    //Imprimimos su valor utilizando la formula (celsius*9/5)+32).
    System.out.println("El valor en grados fahrenheit es "+((celsius*9/5)+32));
    
  }

}

File: primerTrimestre/introduccion_programacion_python/1secuenciales_py/ej5sec.py
# Programa: ej5sec.py
# Propósito: Escribir un programa que convierta un valor dado en grados Fahrenheit a grados Celsius.
# Autor: Antonio Quesada
# Fecha: 13/10/2019.
#
# Variables a usar:
#   * fahrenheit
#   * celsius
#
# Algoritmo:
#   LEER fahrenheit
#   celsius <-- (fahrenheit-32)*5/9
#   ESCRIBIR celsius


# Petición de datos
fahrenheit = float(input("Introduce un valor en grados Fahrenheit: "))
print("-------------------------------------------")

#Cálculos
celsius = (fahrenheit-32)*5/9

# Salida
print("El valor en celsius es", celsius)





